src/api/java/com/teamacronymcoders/epos/api/EposAPI.java

```diff
@@ -18,4 +19,5 @@
     public static final Registry<ISkill> SKILL_REGISTRY = new Registry<>(MissingSkill::new);
     public static final Registry<IPathFeatureProvider> PATH_FEATURE_PROVIDER_REGISTRY =
             new Registry<>(MissingPathFeatureProvider::new);
+    public static final LockRegistry LOCK_REGISTRY = new LockRegistry();
```

Comment:
Why not use a Registry here?

Reply:
Mainly because the information it provides/how it stores it does not really confirm to `Registry<TYPE>`. The biggest of which is the keys of the lock registry are ILockKey's and not strings, and they map to a List of requirements. Additionally, the `LockRegistry`, with the support for "fuzzy keys", which take one key but can match many "key" entries in the LockRegistry.